experimental/python/mobilebert_model.py

```diff
@@ -0,0 +1,79 @@
+import wget
+import argparse
+import os
+import tensorflow as tf
+import tarfile
+import numpy as np
+
+MODEL_DIR = "/tmp/chrome/mobilebert"
+MODEL_URL = "https://storage.googleapis.com/cloud-tpu-checkpoints/mobilebert/mobilebert_squad_savedmodels.tar.gz"
+OUTPUT_DIR = "../android/app/src/main/assets"
+
+def prepare_model(opts):
+  #download the model
+  if not os.path.exists(MODEL_DIR):
+    os.makedirs(MODEL_DIR)
+  saved_model_tar = os.path.join(MODEL_DIR, 'mobilebert_squad_saved.tar.gz')
+  
+  if not os.path.exists(saved_model_tar):
+    wget.download(MODEL_URL, saved_model_tar)
+
+  #extract
+  tar_file = tarfile.open(saved_model_tar)
+  tar_file.extractall(path=MODEL_DIR)
+
+  #convert to tflite
+  model_path = os.path.join(MODEL_DIR, "mobilebert_squad_savedmodels/quant_saved_model")
+  print(model_path)
+  converter = tf.lite.TFLiteConverter.from_saved_model(model_path)
+  tflite_model = converter.convert()
+  with open(os.path.join(opts.out_dir, "text_mobilebert.tflite"), "wb") as f:
+    f.write(tflite_model)
+
+# def evaluate(opts):
+#   print("evaluation")
+#   interpreter = tf.lite.Interpreter(model_path=os.path.join(OUTPUT_DIR, "text_mobilebert.tflite"))
+#   interpreter.allocate_tensors()
+#   input_details = interpreter.get_input_details()
+#   output_details = interpreter.get_output_details()
+#   print(input_details)
```

Comment:
Why do we have this commented block? I guess you would like to keep experimental ?

Reply:
Yeah, let's not review experimental files.